tests/reporters/test_custom_flex_reporter.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+import pytest_asyncio
+from brownie import accounts
+from brownie import SampleFlexReporter
+from telliot_core.apps.core import ChainedAccount
+from telliot_core.apps.core import Contract
+from telliot_core.apps.core import TelliotCore
+from telliot_core.utils.response import ResponseStatus
+
+from telliot_feeds.datafeed import DataFeed
+from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.matic_usd_feed import matic_usd_median_feed
+from telliot_feeds.reporters.custom_flex_reporter import CustomFlexReporter
+from telliot_feeds.utils.log import get_logger
+
+
+logger = get_logger(__name__)
+
+account_fake = accounts.add("023861e2ceee1ea600e43cbd203e9e01ea2ed059ee3326155453a1ed3b1113a9")
+try:
+    account = ChainedAccount("fake").add(
+        "fake", 80001, "023861e2ceee1ea600e43cbd203e9e01ea2ed059ee3326155453a1ed3b1113a9"
+    )
+except Exception:
+    account = ChainedAccount("fake")
+
+
+@pytest.fixture(scope="module", autouse=True)
+def mock_reporter_contract(mock_flex_contract, mock_token_contract, mock_autopay_contract):
+    """mock custom reporter contract"""
+    return account_fake.deploy(
+        SampleFlexReporter,
+        mock_flex_contract.address,
+        mock_autopay_contract.address,
+        mock_token_contract.address,
+        0,
+    )
+
+
+@pytest_asyncio.fixture(scope="function")
+async def custom_reporter(
+    mumbai_test_cfg,
```

Comment:
Why's it using mumbai test cfg? Shouldn't it use the local cfg, or?

Reply:
this is local

Reply:
saw your rinkeby comment, my b